tests/assets/spec-with-functions-no-sources/pipeline.yaml

```diff
@@ -1,8 +1,8 @@
 tasks:
-  - source: my_tasks.raw.functions.function
+  - source: my_tasks2.raw.functions.function
     product: some_file.txt
     name: raw
   
-  - source: my_tasks.clean.functions.function
+  - source: my_tasks2.clean.functions.function
     product: another.txt
```

Comment:
I've changed these assets so that the tests pass, we can maybe revisit it in this ticket? https://github.com/ploomber/ploomber/issues/442

Reply:
I think this is a different problem. see #449

Reply:
but this works for now, thanks!